src/AmesHousing/model_building.py

```
@@ -0,0 +1,116 @@
+import logging
+from abc import ABC, abstractmethod
+from typing import Any
```

Ruff

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)